config/runtime/boot/fastboot.jinja2

```diff
@@ -0,0 +1,39 @@
+- deploy:
+    images:
+      image:
+        url: '{{ node.artifacts.kernel }}'
+      dtb:
+        url: '{{ node.artifacts.dtb }}'
+      ramdisk:
+        url: 'http://storage.kernelci.org/images/rootfs/buildroot/buildroot-baseline/20230703.0/{{ brarch }}/rootfs.cpio.gz'
+        compression: gz
+        format: cpio.newc
+        overlays:
+          lava: true
+     postprocess:
```

Comment:
Attempt to use this template (as described in https://github.com/kernelci/kernelci-pipeline/pull/623#issuecomment-2147008153) results in:

```py
yaml.parser.ParserError: while parsing a block mapping
  in "<unicode string>", line 61, column 5
did not find expected key
  in "<unicode string>", line 68, column 6
```

```suggestion
    postprocess:
```

Aligning indentation allowed me to render TestJob (otherwise [YAML dumping](https://github.com/kernelci/kernelci-core/blob/260d3a5283a8c1a9b07b10fdf8eba37f9e8ad1aa/kernelci/runtime/lava.py#L250) failed)

Reply:
oh, I missed that. Let me fix it quickly.

Reply:
Fixed now.